src/target/riscv/riscv-013.c

```diff
@@ -619,6 +621,7 @@ static int dmi_op_timeout(struct target *target, uint32_t *data_in,
 					LOG_ERROR("Failed %s (NOP) at 0x%x; status=%d", op_name, address,
```

Comment:
```suggestion
					LOG_ERROR("Failed DMI %s (NOP) at 0x%x; status=%d", op_name, address,
```

(There's also one more occurrence above.)

Reply:
Done.